lib/perl/Genome/Config/AnalysisProject/Command/CopyConfig.t

```diff
@@ -51,6 +54,7 @@ my $target_project = Genome::Config::AnalysisProject->create(
 my $cmd = $class->create(
     to_project => $target_project,
     from_project => $source_project,
+    tags => 1,
```

Comment:
I dunno if adding tests is beneficial or not; that's what I meant by "overkill".  I agree they don't seem necessary now.  I'm worried about future changes/growth but YAGNI... maybe.